### PR TITLE
157/add options to api

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
This PR makes changes to the publisher necessary in order to support converting answer options from the Author GraphQL schema into a format that is understood by eq-runner.

### How to review 
Should be able to invoke the Publisher API against the latest GraphQL API and observe that valid survey runner output is produced.

### CODE REVIEW

**Comment by samiwel** [comment]
[Add options to Radio and Checkbox answers in the API (8)](https://trello.com/c/0iYDNuQ0/157-add-options-to-radio-and-checkbox-answers-in-the-api-8)

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8"
+  - "7"
```
Comment (WickyNilliams) [inline .travis.yml]: what's the reason for dropping the node version?

Comment (jonnyshaw89) [inline .travis.yml]: travis doesn't properly support 8 yet?

Comment (WickyNilliams) [inline .travis.yml]: Ah fair enough

**Comment by samiwel** [comment]
This PR should be fine to merge. The build failure is a result of how I've named my branch, with a "/".

**Pull request merged**